cost/google/idle_compute_instances/google_idle_compute_instances.pt

```diff
@@ -357,9 +360,12 @@ policy "pol_utilization" do
     escalate $terminate_resources
     export do
       resource_level true
-      field "projectId" do
+      field "projectID" do
```

Comment:
this policy is using `projectID` field whereas other Google policies are using `accountID` field? Are these meant to represent the same thing? Seems a little inconsistent if so

Reply:
Just seen conversation below, I believe this is being done so data isn't scraped? @lawrethao @jim-slattery-rs